projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/dimension/scd1/02-handle-quality-checks.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+
+from vdk.api.job_input import IJobInput
+
+SQL_FILES_FOLDER = (
+    os.path.dirname(os.path.abspath(__file__)) + "/02-requisite-sql-scripts"
+)
+
+
+def run(job_input: IJobInput):
+    job_arguments = job_input.get_arguments()
+
+    check = job_arguments.get("check")
+    target_schema = job_arguments.get("target_schema")
+    target_table = job_arguments.get("target_table")
+    staging_schema = job_arguments.get("staging_schema")
+    insert_query = get_query("02-insert-into-target.sql")
+
+    if check:
+        if not staging_schema:
+            raise Exception(
```

Comment:
It's considered bad practice to use the base Exception class . 

It's better to use specific exception. For Illegal arguments ValueError is a frequently used

Reply:
Changed that.